subtests/docker_cli/load/load.py

```diff
@@ -26,7 +28,7 @@ class load(subtest.Subtest):
     def initialize(self):
         super(load, self).initialize()
         self.stuff['di'] = di = DockerImages(self)
-        img_id = self.config['test_id'].strip()
+        img_id = self._test_id
```

Comment:
No, see the new _test_id() below. This is a temporary coexistence hack for 1.9 and 1.10. I've tried to come up with a better way to do this, but can't think of anything.